src/native/fs/fs_napi.cpp

```diff
@@ -182,6 +208,7 @@ struct Stat : public FSWorker
     }
     virtual void Work()
     {
+        DBG1("FS::Stat::Work: " << DVAL(_path));
```

Comment:
If you want all fsworkers to log1 then add a print on the base class in Execute...

Reply:
But the base class prints the action name using _desc... Not sure why you want to leave all those calls when you already have it?

Reply:
where do we print the action name? I never seen any print like it. in _desc we have just the path: ```FS::FSWorker::Begin: _path=/nsfs``` and that's for lucky few that has begin.

Reply:
So add the action name to Begin, and add missing calls to Begin from FSWorkers that did not add it.

Reply:
#6497